### PR TITLE
Kp moving containers from warp pd 2185

### PR DESCRIPTION
**Pull request by kevinpalis**
This cleans up all the containers from WARP, deprecates ones that we don't use anymore, and moves everything else to WARP-tools. The corresponding PR in WARP is here: https://github.com/broadinstitute/warp/pull/987

### CODE REVIEW

File: build-indices_wt/add-introns-to-gtf.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
```
Comment (ekiernan) [inline build-indices_wt/add-introns-to-gtf.py]: @aawdeh Would you be able to please confirm that this add-introns script has your latest changes in it? This is the script that adds intronic regions to the GTF.

File: build-indices_wt/modify_gtf.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
+import argparse
```
Comment (ekiernan) [inline build-indices_wt/modify_gtf.py]: @kevinpalis This is not the same version of the script that's on the WARP develop branch. The WARP develop branch has the correct version. https://github.com/broadinstitute/warp/blob/develop/dockers/skylab/build-indices/modify_gtf.py

Comment (kevinpalis) [reply build-indices_wt/modify_gtf.py]: Hi @ekiernan - sorry for the confusion, I forgot to delete this. The actual directory of build-indices is this one: https://github.com/broadinstitute/warp-tools/tree/kp_moving_containers_from_warp_PD-2185/build-indices

File: build-indices_wt/add-introns-to-gtf.py (hunk 1 above)
Comment (aawdeh) [reply build-indices_wt/add-introns-to-gtf.py]: I just checked add `add-introns-to-gtf.py` and it does have the latest changes.

**Review by ekiernan — APPROVED** [review]

**Pull request merged**